From 385f5f5e9722eefd287771008f90475fefe09b27 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 21 Jun 2013 15:56:45 +0200
Subject: [PATCH] journald: bump the journal per-unit ratelimit defaults

Too many people kept hitting them, so let's increase the limits a bit.

https://bugzilla.redhat.com/show_bug.cgi?id=965803
---
 src/journal/journald-server.c | 4 ++--
 src/journal/journald.conf     | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/journal/journald-server.c b/src/journal/journald-server.c
index de8ac68ad0..24c52271d7 100644
--- a/src/journal/journald-server.c
+++ b/src/journal/journald-server.c
@@ -65,8 +65,8 @@
 #define USER_JOURNALS_MAX 1024
 
 #define DEFAULT_SYNC_INTERVAL_USEC (5*USEC_PER_MINUTE)
-#define DEFAULT_RATE_LIMIT_INTERVAL (10*USEC_PER_SEC)
-#define DEFAULT_RATE_LIMIT_BURST 200
+#define DEFAULT_RATE_LIMIT_INTERVAL (30*USEC_PER_SEC)
+#define DEFAULT_RATE_LIMIT_BURST 1000
 
 #define RECHECK_AVAILABLE_SPACE_USEC (30*USEC_PER_SEC)
 
diff --git a/src/journal/journald.conf b/src/journal/journald.conf
index 5410477201..54f6833a17 100644
--- a/src/journal/journald.conf
+++ b/src/journal/journald.conf
@@ -13,8 +13,8 @@
 #Seal=yes
 #SplitMode=login
 #SyncIntervalSec=5m
-#RateLimitInterval=10s
-#RateLimitBurst=200
+#RateLimitInterval=30s
+#RateLimitBurst=1000
 #SystemMaxUse=
 #SystemKeepFree=
 #SystemMaxFileSize=
